### ISSUE TITLE
training time and test data set

### ISSUE THREAD

**Issue by lfxuan** [xamyzhao/brainstorm#18]
Hi,Amy!
Thank you for your excellent work. Now I'm using your code to train a segmentation network. There are two questions I want to ask you. (1) During the segmentation training, the number of steps is set to 10000, and a Tesla V100 PCIe 32GB GPU card is used to execute" python main.py fss --gpu 0 
-- data mri-100unlabeled -- aug_tm".This process takes about 57 hours, which is too long.So, what kind of GPU cards do you use for training? How long does 100000 steps take?
 (2) whether the verification set provided by the project can evaluate the performance of the segmented network?Training the segmented network, setting 10000 steps, using the verification set test, the segmentation result of "- -aug _ sas" is slightly worse than that of the directly trained the segmentation network. In this case, I don't know whether it's a step number problem or a verification set problem?Can you provide a test set?

**Comment by xamyzhao** [comment]
(1) I typically train the network until the validation loss plateaus; the 10000 is just set as a maximum training time but in my experience 3000-5000 epochs is sufficient. I used a Titan X 12GB GPU. 
(2) The validation folder in this project is intended as a sample as it only includes a single scan. We collected the test set used in the paper from multiple public datasets, but unfortunately it would be complicated for us to re-host their data in this repo. I would recommend referring to the paper for the datasets that we used.
